Content.Server/_DV/CosmicCult/Components/CosmicFinaleComponent.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+
+namespace Content.Server._DV.CosmicCult.Components;
+
+[RegisterComponent, AutoGenerateComponentPause]
+public sealed partial class CosmicFinaleComponent : Component
+{
+    [DataField]
+    public FinaleState CurrentState = FinaleState.Unavailable;
+
+    [DataField]
+    public bool FinaleDelayStarted = false;
+
+    [DataField]
+    public bool FinaleActive = false;
+
+    [DataField]
+    public bool Occupied = false;
```

Comment:
Consistency in the file

```suggestion
    [DataField] public FinaleState CurrentState = FinaleState.Unavailable;

    [DataField] public bool FinaleDelayStarted;

    [DataField] public bool FinaleActive;

    [DataField] public bool Occupied;
```